src/components/timers/Countdown.tsx

```diff
@@ -0,0 +1,47 @@
+import { useContext } from 'react';
+import type React from 'react';
+import { useCountdownTimer } from '../../hooks/useCountdownTimer';
+import { useTimerSync } from '../../hooks/useTimerSync';
+import type { TimerProps } from '../../utils/TimerProps';
+import { DisplayRepsForText } from '../../utils/helpers';
+import { TimeDisplay, Timer, TimerContainer, TimerTitle } from '../../utils/styles';
+import { Button, Buttons } from '../../utils/styles';
+import { TimersContext } from '../../views/TimerProvider';
+
+const Countdown: React.FC<TimerProps> = ({ repInput, totalSeconds, isActive, isCurrent, onFinish, isFinished }) => {
+    const { secondsPassed, setSecondsPassed, fastforward } = useCountdownTimer(totalSeconds, isActive, onFinish, Number(repInput), 0);
```

Comment:
Be careful with these function signatures. When a function receives many arguments and they're not named, its hard to remember what is what. For example here, what is the last argument `0`?